pkg/server/backend_manager.go

```diff
@@ -27,6 +28,29 @@ import (
 	"sigs.k8s.io/apiserver-network-proxy/proto/agent"
 )
 
+type ProxyStrategy string
+
+const (
+	ProxyStrategyDestHost ProxyStrategy = "destHost"
+	ProxyStrategyDefault  ProxyStrategy = "default"
+)
+
+// GenProxyStrategiesFromStr generates the list of proxy strategies from the
+// comma-seperated string, i.e., destHost,defualt.
+func GenProxyStrategiesFromStr(proxyStrategies string) []ProxyStrategy {
+	var ps []ProxyStrategy
+	strs := strings.Split(proxyStrategies, ",")
+	for _, s := range strs {
+		switch s {
+		case string(ProxyStrategyDestHost):
+			ps = append(ps, ProxyStrategyDestHost)
+		default:
+		}
+	}
+	// always append the default backendmanager as the last one
+	return append(ps, ProxyStrategyDefault)
```

Comment:
We document above that you can explicitly place default in the strategies config. To me that means if I don't include it then it should be absent (which seems correct). However that means it should be in the switch statement. It is then possible to not have a default strategy if that is what you want.

Reply:
That makes sense. I have removed the default strategy. And the `DefaultBackendManager` will always be added to the end of `BM` list when creating a new proxy server